TLM/TLM/Resources/lang.txt

```diff
@@ -59,6 +59,7 @@ All_vehicles_may_ignore_lane_arrows All vehicles may ignore lane arrows
 Busses_may_ignore_lane_arrows Buses may ignore lane arrows
 Turn_on_red_enabled_by_default Turn on red enabled at junctions by default
 Turn_on_red Vehicles may turn at red traffic lights
+Disable_Lane_Change_At_Edge Disable Lane Change At Edge of the map
```

Comment:
move to end-of-file please and add a dummy translation (English) for every other language